bin/git-co-upstream-pr

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+pull_request_number=$1
+local_branch_name=$2
```

Comment:
If these aren't passed, we should probably error helpfully:

```sh
if [ -z "$pull_request_number" -o -z "$local_branch_name" ]; then
  echo "usage: git co-upstream-pr <pull_request_number> <local_branch_name>"
  echo 1
fi
```

Reply:
OH, good idea! This entire script writing is so new to me, this totally escaped me. Added this check and instead of `echo 1`, I think we can exit here instead of proceeding to the next check. Did that in e0d610aba0edd1bc80862325a015f81856fb542b